src/measure_theory/card_measurable_space.lean

```diff
@@ -0,0 +1,141 @@
+/-
+Copyright (c) 2022 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import measure_theory.measurable_space_def
+import set_theory.continuum
+import set_theory.cofinality
+
+/-!
+# Cardinal of sigma-algebras
+
+If a sigma-algebra is generated by a set of sets `s` whose cardinality is at most the continuum,
+then the sigma-algebra satisfies the same cardinality bound. This is stated in
+`measurable_space.cardinal_generate_measurable_le` and
+`measurable_space.cardinal_measurable_set_le`.
+
+For the proof, we rely on an explicit inductive construction of the sigma-algebra generated by
+`s` (instead of the inductive predicate `generate_measurable`). This transfinite inductive
+construction is parameterized by an ordinal `< ℵ₁`, and the cardinality bound is preserved along
+each step of the construction.
+-/
+
+universe u
+variables {α : Type u}
+
+open_locale cardinal
+open cardinal set
+
+local notation a `<₁` b := (aleph 1 : cardinal.{u}).ord.out.r a b
```

Comment:
Is the universe polymorphism really necessary? Maybe I'm wrong but I believe everything should work just fine if you just took this in universe 0.

Reply:
My end goal is to prove a result on sigma-algebras in alpha, which lives in universe `u`. So things work smoother if every cardinal computation is made in universe `u` -- if I take this in universe 0, I have to add a bunch of ulifts all over the place.

Reply:
That's fair.